Realm.Shared/native/NativeException.cs

```diff
@@ -29,22 +29,22 @@ internal unsafe struct NativeException
         public sbyte* messageBytes;
         public IntPtr messageLength;
 
-        internal Exception Convert()
+        internal Exception Convert(Func<RealmExceptionCodes, Exception> overrider)
```

Comment:
I am not sure exactly why I felt it was different here. It was a conscious, though hesitant, choice. But since I don't really have a reason, I'll change it :-)